test/svg-sprite/minimal-configuration/modes/css.test.js

```diff
@@ -97,25 +97,19 @@ describe('testing minimal config', () => {
             // Horizontal layout
             it('horizontal layout', async() => {
                 expect.hasAssertions();
-                await expect(path.join(tmpPath, 'css/svg', svg.horizontal)).toBeVisuallyEqual(
-                    path.join(paths.expectations, `png/css.horizontal${testConfig.namespace}.png`)
-                );
+                await expect(path.join(tmpPath, 'css/svg', svg.horizontal)).toBeVisuallyEqual(path.join(paths.expectations, `png/css.horizontal${testConfig.namespace}.png`));
```

Comment:
@Kreeg let's split such cases in 2 variables, like `input`, `expected` or something like that in all cases. It should make things more clear and we can get rid of the long lines :)